### DISCUSSION TITLE
I have the github student developer account

### DISCUSSION THREAD

**Discussion by KeithPrinkey-ops** [Labs64/NetLicensing-Community#115] [Questions] [answered]
I have the github student developer account and tried redeeming my free student license but the link takes me to the homepage? How do I enable the student license?

**Comment by r-brown** [comment, marked as answer]
Hello @KeithPrinkey-ops 
GitHub Students program is discontinued.

NetLicensing is supporting students, teachers and educational organizations though.
Please [contact us](https://netlicensing.io/contact/) providing your student status, school, etc. and we'll assign you Students Plan after the verification.

**Comment by KeithPrinkey-ops** [comment]
My school email address is ***@***.*** and i attend Columbia
College of Missouri.

What other information do you need?

**Comment by r-brown** [comment]
> My school email address is ***@***.*** and i attend Columbia College of Missouri. What other information do you need?

Please check _NetLicensing Education_ program requirements at https://github.com/Labs64/NetLicensing-Community/issues/50#issuecomment-1133587275
